lang/shared/src/main/scala/com/wavesplatform/lang/v1/evaluator/ctx/impl/waves/Functions.scala

```diff
@@ -534,4 +534,34 @@ object Functions {
           }
       }
     }
+
+  val simplifiedIssueActionConstructor: BaseFunction[Environment] =
+    NativeFunction(
+      "Issue", 1, SIMPLIFIED_ISSUE_ACTION_CONSTRUCTOR, issueActionType,
+      FieldNames.IssueName -> STRING,
+      FieldNames.IssueDescription -> STRING,
+      FieldNames.IssueQuantity -> LONG,
+      FieldNames.IssueDecimals -> LONG,
+      FieldNames.IssueIsReissuable -> BOOLEAN,
+    ) {
+      args =>
+        val typedArgs = (issueActionType.fields.map(_._1) zip (args ::: List(unit, CONST_LONG(0)))).toMap
+        Right(CaseObj(issueActionType, typedArgs))
+    }
+
+  val detailedIssueActionConstructor: BaseFunction[Environment] =
```

Comment:
Is the origin constructor of the Issue structure removed from the context?

Reply:
Not removed but overlapped